Pg limitations 1.1.0 (#145)
percona/everest-doc: docs/reference/known_limitations.md

## Backups

Let's delve into the limitations of on-demand backups in Percona Everest. 

### PostgreSQL limitations for on-demand backups

- When attempting to delete a PostgreSQL database that contains backups created with Everest versions older than 1.0.0, the database may become stuck in the **Deleting** state. 

    **Workaround**: To prevent this, manually delete any backups created with versions prior to 1.0.0 by using the **Delete** action on the **Backups** page before deleting the database.

> - You cannot change the bucket name for a specific backup storage. Doing so will make any backups taken for that bucket unusable.

Reviewer: > You cannot change the bucket name

the bucket name _and the region_

Author: Fixed

> - You cannot change the bucket name and region for a specific backup storage. Doing so will make any backups taken for that bucket unusable.

Reviewer 2: @oksana-grishchenko Haven't we enforced this in the UI/API already?

Reviewer 3: @recharte from the UI side, we did.

Reviewer 2: In this case I don't think we should mention the last sentence. If we don't allow it to be change the consequences never happen.
```suggestion
- You cannot change the bucket name and region for a specific backup storage.
```

Reviewer 2: Actually I'm not sure anymore :sweat_smile: , I leave it up to Rasika... Keeping the info of what would happen if we did allow it is also fine. You can ignore this comment.

- You can create on-demand backups using only the first three backup storages if there are more than three backup storages.

>     For example, if you create backup storages everest-1, everest-2, everest-3, and everest-4 sequentially, you can only use everest-1, everest-2, and everest-3 for on-demand backups.

Reviewer 3: Maybe:

> You can create on-demand backups and schedules using only the first three backups storages if there are more than three backups storages in use across on-demand backups and schedules.
For example, if you create two schedules using backups storages everest-1 and everest-2 and an on-demand backup using backup storage everest-3, you can only use one of those storages to create an on-demand backups or a schedule."

Author: @fabio-silva I find this confusing. Could you please help me understand this.

Author: @fabio-silva WDYT about this:

If more than three backup storages are created, you can only use the first three storages across on-demand backups and schedules.

For example, if you have created two schedules using backup storage bucket-1 and bucket-2, and an on-demand backup using backup storage bucket-3, you can only utilize one of these three backup storages to create the next on-demand backup or a schedule.

Author: Fixed

> - If you create more than three backup storages, you can use any of the existing storages across on-demand backups and schedules, as long as the total number of storages in use (by existing on-demand backups and schedules) does not exceed three.

Reviewer 2: This first part seems redundant to me, what do you this of the following suggestion?
```suggestion
- You can use any of the existing backup storages across on-demand backups and schedules, as long as the total number of storages in use (by existing on-demand backups and schedules) does not exceed three.
```

>     If you have created two schedules using backup storage bucket-1 and bucket-2, and an on-demand backup using backup storage bucket-3, you can only utilize one of these three backup storages to create the next on-demand backup or a schedule.

Reviewer 2: Can we format the buckets names differently? Using italic for example?

Author: Done

## Scheduled backups

Let's explore the constraints of scheduled backups in Percona Everest.

### PostgreSQL Limitations for schedules

Due to PostgreSQL limitations, the following functionality is unavailable for PostgreSQL:

- Modifying the storage location in existing schedules
- Using the same backup storage for different schedules
- Creating more than three schedules for PostgreSQL

> - Creating a schedule that uses the fourth backup storage when three backup storages are already in use.

Reviewer: the last two are not exactly true, the truth is: 

- Using more than 3 different storages in total. The total number of used storages includes the storages used in the existing backups and schedules.

> - Using more than three different backup storages in total, including those used in existing backup schedules.

Reviewer 2: Since we are in the schedules section wouldn't it make more sense to refer to the on-demand ones instead?
```suggestion
- Using more than three different backup storages in total, including those used in existing on-demand backups.
```
